save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ExecutionView.kt

```diff
@@ -39,24 +49,48 @@ class ExecutionView : RComponent<ExecutionProps, RState>() {
                         +"${it.row.index}"
                     }
                 }
-                column(id = "name", header = "Test name") {
+                column(id = "startTime", header = "Start time") {
                     td {
-                        +"${it.value["name"]}"
+                        +"${it.value.startTimeSeconds?.let { Instant.fromEpochSeconds(it, 0) }}"
                     }
                 }
                 column(id = "status", header = "Status") {
                     td {
-                        +"${it.value["result"]}"
+                        +"${it.value.status}"
+                    }
+                }
+                column(id = "path", header = "Test file path") {
+                    td {
+                        +it.value.filePath
                     }
                 }
+            },
+            useServerPaging = true,
+            getPageCount = { pageSize ->
+                val count: Int = get(
+                    url = "${window.location.origin}/testExecutionsCount?executionId=${props.executionId}",
+                    headers = Headers().also {
+                        it.set("Accept", "application/json")
+                    },
+                )
+                    .json()
+                    .await()
+                    .unsafeCast<Int>()
+                count / pageSize + 1
             }
-        ) {
-            arrayOf(
-                json("name" to "test 1", "result" to "passed"),
-                json("name" to "test 2", "result" to "failed"),
-                json("name" to "test 3", "result" to "skipped"),
-                json("name" to "test 4", "result" to "failed"),
+        ) { page, size ->
```

Comment:
Maybe left here comment with stub to load data, because now it can't load data?

Reply:
What do you mean?) The whole purpose of this PR is to add data loading here